ui/raidboss/data/07-dt/raid/r4n.txt

```diff
@@ -1,8 +1,123 @@
 ### Arcadion (R4N): AAC Light-heavyweight M4
 
+# -ii 90EE 92B3 92B5 92B6 98EA 8A07 8D36 8F97 92C3
+# -it "Wicked Thunder"
+# -p 92B1:1004.7
+
+# 90EE - normal autoattack
+# 8F97 - autoattack when wings are out?
+# 92C9 - teleport
+# 92A7 - Something to do with the tp north, maybe what triggers gun?
+# 92AB - ライトロール - "Right Roll" - boss shooting her right/player left 3/4ths
+# 92AC - レフトロール - "Left Roll" - boss shooting her left/player right 3/4ths
+
+# 92BC/92BD/92BE/92BF - boss cleaves
+# 9A05/9A0F - clone cleaves
+
+
 hideall "--Reset--"
 hideall "--sync--"
 
 0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
 
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+6.9 "--sync--" StartsUsing { id: "92C7", source: "Wicked Thunder" } window 10,10
+11.6 "Wrath of Zeus" Ability { id: "92C7", source: "Wicked Thunder" }
+24.7 "Sidewise Spark" Ability { id: ["92BC", "92BD", "92BE", "92BF"], source: "Wicked Thunder" }
+30.8 "--middle--" Ability { id: "92C9", source: "Wicked Thunder" }
+35.1 "Shadows' Sabbath" Ability { id: "9BBF", source: "Wicked Thunder" }
+44.3 "Sidewise Spark" Ability { id: ["9A05", "9A0F"], source: "Wicked Replica" }
```

Comment:
For the `Sidewise Spark`s coming from the replicas, would it be beneficial to add a parenthetical here to note that, e.g. `(clone)`, to differentiate them from boss attacks by the same name?